src/postgres/src/backend/executor/ybcModifyTable.c

```diff
@@ -342,7 +347,7 @@ static Oid YBCApplyInsertRow(
 	if (IsCatalogRelation(rel))
 	{
 		MarkCurrentCommandUsed();
-		CacheInvalidateHeapTuple(rel, tuple, NULL);
+		CacheInvalidateHeapTuple(rel, slot->tts_tuple, NULL);
```

Comment:
If I'm not missing something, local variable tuple is defined here and equal to slot->tts_tuple